.buildkite/pipeline.yml

```diff
@@ -37,5 +32,14 @@ steps:
       bundle install
       bundle exec fastlane test_without_building name:WordPressUnitTests try_count:3
     depends_on: "build"
+
+  #################
+  # Lint Translations
+  #################
+  - label: "🧹 Lint Translations"
+    command: "gplint /workdir/WordPress/Resources/AppStoreStrings.po"
```

Comment:
Was looking at [your docker image](https://github.com/jkmassel/glotpress-linter), and 🤯  using PHP and require `/wordpress/wp-includes/pomo/po.php` to use our own GlotPress's `$po->import_from_file(…)` to lint… 💥  would never have thought of that! 👌 👏

Reply:
+1